src/app/xapian/searchservice.ts

```diff
@@ -103,6 +103,7 @@ export class SearchService {
   localdir: string;
   partitionsdir: string;
 
+  isExpired: boolean = false;
```

Comment:
If you feel like it, it might be a good idea to add that as a method in `RunboxMe` instead since there's seems to be a lot of `me.account_status === 'expired'`